docs/apis-tools/administration-api/tutorial.md

```diff
@@ -33,6 +33,10 @@ To set up your credentials, create an `.env` file which will be protected by the
 
 These keys will be consumed by the `auth.js` file to execute the OAuth protocol, and should be saved when you generate your client credentials in [prerequisites](#prerequisites).
 
+:::note Can't find your environment variables?
```

Comment:
```suggestion
:::tip Can't find your environment variables?
```

non-blocking: Are these blocks more of a tip than a note?

Reply:
Yes I agree!